build/docker/Dockerfile

```diff
@@ -9,7 +9,7 @@ COPY ./ ./
 RUN dotnet build --configuration Release
 RUN dotnet publish "./src/Microsoft.Health.Fhir.${FHIR_VERSION}.Web/Microsoft.Health.Fhir.${FHIR_VERSION}.Web.csproj" -c Release -o "/build" --no-build
 
-FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-alpine AS runtime
+FROM mcr.microsoft.com/dotnet/core/aspnet:3.1.6-alpine AS runtime
```

Comment:
This doesn't look like it is a supported tag. I don't think we need to change this tag.
https://hub.docker.com/_/microsoft-dotnet-core-aspnet
Linux amd64 tags

Reply:
There is a version specific alpine tag, however I do see that it has already moved to 3.1.7.  You are correct that previously we simply had the 3.1 tag, so I reverted this version specific change.